main.go

```diff
@@ -119,26 +131,26 @@ func getJobsData(jobsUrl string) {
 	if err != nil {
 		panic(err)
 	}
-	//fmt.Println(jobsRes.TotalCount)
+	fmt.Println(jobsRes.TotalCount)
 
-	for i, _ := range jobsRes.Jobs {
+	for i := range jobsRes.Jobs {
 		job := &jobsRes.Jobs[i]
-		//fmt.Println("=================================")
-		//fmt.Println("Job Name: ", job.Name)
+		fmt.Println("=================================")
+		fmt.Println("Job Name: ", job.Name)
 		//fmt.Println("Job StartTime: ", job.StartTime)
 		// Calculate job runtime
 		runTime := getRuntime(job.StartTime, job.CompletedTime)
```

Comment:
If I run unauthenticated I get this error
```
$ go run main.go 
Connected to MongoDB successfully!
jobs collection dropped...
6
=================================
Job Name:  build-v2-beta (windows-2022)
2023/06/22 09:02:32 parsing time "" as "2006-01-02T15:04:05Z07:00": cannot parse "" as "2006"
exit status 1
```
It's a strange error so it's hard to pinpoint exactly where it's coming from, but I think it's if I try to parse time when the field is empty